src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
```suggestion
public interface FileReader {
```
no need to add Service to the name of the class if naming is OK without it